executor/runcexecutor/executor.go

```diff
@@ -22,15 +22,38 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+type Opt struct {
+	Root     string
+	Commands []string
```

Comment:
nit: On first glance it looked as if it were arguments to me. How about s/Commands/CommandCandidates/?

Reply:
updated